pipelines/_ingredients/DRP-full.yaml

```diff
@@ -58,7 +54,16 @@ tasks:
   writeRecalibratedSourceTable:
     class: lsst.pipe.tasks.postprocess.WriteRecalibratedSourceTableTask
     config:
-      connections.outputCatalog: source
+      # Use v2 name instead of old v1 name ('source') to avoid clash with v2
+      # final data product name.
+      connections.outputCatalog: recalibrated_star_unstandardized
+  transformSourceTable:
+    class: lsst.pipe.tasks.postprocess.TransformSourceTableTask
+    config:
+      # Use v2 name instead of old v1 name ('source') to avoid clash with v2
+      # final data product name.
+      connections.inputCatalog: recalibrated_star_unstandardized
+      functorFile: $PIPE_TASKS_DIR/schemas/Source.yaml
```

Comment:
Curious if there's a reason why this is moved further down. The ordering doesn't matter here, right?